src/assets/simp/build/simp.spec

```diff
@@ -133,6 +133,14 @@ Obsoletes:  pupmod-puppetlabs-docker  <= 3.5.0
 Obsoletes:  pupmod-simp-simp_docker  <= 0.2.1
 Obsoletes:  pupmod-simp-journald   <= 1.1.0
 
+#  These lines are required for upgrades from 6.5.0 to 6.6.0
```

Comment:
We shouldn't need to do any of this past 6.4.0.

Packages are now inert by default so, while they may be dangling, there's no reason to make a mess of `Obsoletes` that we may have to redact in the future.

I may be missing something, could you point at an example where this causes issues?